src/api/method/get_multiple_new_address_proofs.rs

```diff
@@ -13,12 +13,11 @@ use crate::common::typedefs::context::Context;
 use crate::common::typedefs::hash::Hash;
 use crate::common::typedefs::serializable_pubkey::SerializablePubkey;
 use crate::ingester::parser::tree_info::TreeInfo;
-use crate::ingester::persist::indexed_merkle_tree::{
-    get_exclusion_range_with_proof_v1, get_exclusion_range_with_proof_v2,
-};
+use crate::ingester::persist::indexed_merkle_tree::get_multiple_exclusion_ranges_with_proofs_v2;
+use std::collections::HashMap;
 
-pub const FORESTER_MAX_ADDRESSES: usize = 500;
-pub const MAX_ADDRESSES: usize = 50;
+pub const FORESTER_MAX_ADDRESSES: usize = 4000;
+pub const MAX_ADDRESSES: usize = 1000;
```

Comment:
can we do higher limits?
we should be able to send 5 or 6 proofs per tx with lookup tables instead of the current 4.
```suggestion
pub const FORESTER_MAX_ADDRESSES: usize = 5000;
pub const MAX_ADDRESSES: usize = 1500;
```

Reply:
kk

Reply:
same for max addresses?